front/components/landing/landing.html

```diff
@@ -1 +1,2 @@
+
 <featured-Brands-Directive></featured-Brands-Directive>
```

Comment:
you don't need to upper case the directive usage.  When you hyphenate, everything should be lowercase.

featured-brands-directvie instead of feature-Brands-Directive